cli/src/cmd/run.rs

```diff
@@ -123,7 +137,9 @@ impl Cmd for RunArgs {
                 })
                 .collect();
 
-            let calls = evm.debug_calls();
+            // let calls = evm.debug_calls();
+            // TODO: How should we instantiate this now that we don't have an EVM?
```

Comment:
we will need to return the `DebugArena` in the `TestResult` now like we do for traces. This is probably how it always should have been done anyway